src/types/products.ts

```diff
@@ -26,7 +26,13 @@ interface Product extends ProductMeta {
 
 interface ProductData extends Product {
   basePaths: string[]
-  navigationHeaderItems: NavigationHeaderItem[]
+  navigationHeaderItems: {
+    [key: string]: {
+      icon: NavigationHeaderItem['icon']
+      pathSuffix: string
+      label: NavigationHeaderItem['label']
+    }[]
+  }
```

Comment:
Would like to make this code look nicer, but not sure what to name an interface for the following at the moment. Any ideas welcome! 😊

```tsx
interface SuperCoolInterface {
  icon: NavigationHeaderItem['icon']
  pathSuffix: string
  label: NavigationHeaderItem['label']
}
```